### ISSUE TITLE
failed to install lcms2 on OSX 10.11

### ISSUE THREAD

**Issue by newser1** [Homebrew/homebrew-science#3090]
failed to install Octave on OSX 10.11 with msg: Download failed: https://downloads.sourceforge.net/project/lcms/lcms/2.7/lcms2-2.7.tar.gz. Sourceforge lists a different path to the file http://sourceforge.net/projects/lcms/files/lcms/2.7/lcms2-2.7.tar.gz/download, with the "files" subdirectory missing from the gitHub path, how can I correct this?

**Comment by dpo** [comment]
It works for me. What do these commands say:
```
rm /Library/Caches/Homebrew/little-cms2-2.*
brew install lcms2
```
If you get an error, please post the output of `brew doctor`.

**Comment by dpo** [comment]
See #3091.

**Comment by newser1** [comment]
1) Received msg(rm: /Library/Caches/Homebrew/little-cms2-2.*: No such file or directory) twice, both before and after running (brew install lcms2).

2) Only firewall running is one supplied by my Mac OSX10.11 under system preferences
Why do the other dependency downloads up to this pt in the install do fine?

3) running bash-3.2$ brew doctor
Your system is ready to brew.

 What should I do next?

**Comment by newser1** [comment]
Sorry. I'm a first time user, made several attempts and sent the same issue twice and posted this email as a comment on GitHub also.

As for my problem,Received (rm: /Library/Caches/Homebrew/little-cms2-2.*: No such file or directory) twice, both before and after running (brew install lcms2).
Only firewall running is one supplied by my OSX10.11Why do the other dependencies download without problems up to this pt in the install?
  running brew doctor:bash-3.2$ brew doctor
Your system is ready to brew.
 What should I do next?     


    On Friday, December 4, 2015 3:58 PM, Dominique <notifications@github.com> wrote:
 

 Closed #3090.—
